src/common/reducers/entities.js

```diff
@@ -0,0 +1,62 @@
+import merge from 'lodash/merge';
+
+import * as ActionTypes from '../actions/repositories';
+
+// export only for testing
+export function repository(state={
+  __isSelected: false,
```

Comment:
@bartaz further to comments on prefixing (https://github.com/canonical-ols/build.snapcraft.io/pull/541#discussion_r109158954), I was thinking over the weekend that if we whitelisted props from the api in the schema (normalize) we'd have a number of benefits:

- control over properties, no clobbering
- smaller api from server to client
- invariant api, so no chance of performance variation because source adds more data (that we don't use)

and is such, `isSelected` et al no longer need a prefix